spec/spec_helper.rb

```diff
@@ -41,6 +41,16 @@
   'nbHits'
 ].freeze
 
+FINITE_PAGINATED_SEARCH_RESPONSE_KEYS = [
+  'hits',
+  'query',
+  'processingTimeMs',
+  'hitsPerPage',
+  'page',
+  'totalPages',
+  'totalHits',
+].freeze
```

Comment:
Why isn't `hits_per_page` in this list? Sorry if these are dumb questions 🙈

Reply:
It is because these are the keys that the Meilisearch server will send to us, and the server only has camelCased ones. :D 

Don't worry, no question is dumb, feel free to ask anything!